src/utils/decamelize.ts

```diff
@@ -0,0 +1,26 @@
+// Simplified copy of https://github.com/sindresorhus/decamelize/blob/5deb8c5b88c3dbc93ef0d68faa0fc45cf98cad9d/index.js
+const REPLACEMENT = '$1_$2';
+
+export function decamelize(text: string): string {
```

Comment:
I recommend adding some jsdocs to the methods, so that you don't have to look at the implementation to know what it does.

Reply:
thx for the hint :+1: 
will do that